src/convertToObject.js

```diff
@@ -11,8 +11,26 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const values = styles.map(style => {
+    const value = style.slice(style.indexOf(':') + 1, style.length).trim();
+
+    return value;
+  });
+  const properties = styles.map(style => {
+    const property = style.slice(style.indexOf(''), style.indexOf(':')).trim();
+
+    return property;
+  });
```

Comment:
You did the same operations for values and for properties
Try to create one array with elements that also contains an array [property, value]